setup.py

```diff
@@ -13,38 +13,6 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-import os
-import sys
+from setuptools import setup
 
-from setuptools import (
-    find_packages,
-    setup,
-)
-
-assert sys.version_info >= (3,), "Python 3 is required"
-
-
-VERSION = "0.0.1"
-
-datafiles = [
-    (d, [os.path.join(d, f) for f in files])
-    for d, folders, files in os.walk("data")
-]
-
-setup(
-    name="snappy-device-agents",
-    version=VERSION,
-    description=(
-        "Device agents scripts for provisioning and running "
-        "tests on Snappy devices"
-    ),
-    author="Snappy Device Agents Developers",
-    author_email="paul.larson@canonical.com",
-    url="https://launchpad.net/snappy-device-agents",
-    license="GPLv3",
```

Comment:
do you want not to specify this information in the pyproject.toml file?

Reply:
As above, none of this is really useful here. I did add the license at least. Metadata is great for widely used packages on pypi, but we don't really have a reason to push it there.